myst_nbs/diagnostics_and_criticism/model_averaging.myst.md

```diff
@@ -6,11 +6,20 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python PyMC3 (Dev)
+  display_name: Python 3 (ipykernel)
   language: python
-  name: pymc3-dev-py38
+  name: python3
 ---
 
+(model_averaging)=
+# Model Averaging
+
+:::{post} Aug 2022
+:tags: diagnostics, model selection, model comparison
```

Comment:
I would remove the diagnostics and model selection tags, keep model comparison and "model averaging" (which should be added.

To me, model comparison is kind of a wide field/superset inside of which you can do predictive/cross validation based model comparison, hypotheses based model comparison, model selection, model averaging... Therefore, if we update the tags like suggested above, people can browse the model comparison tag for all different techniches (it has bayes factors there for example) or model averaging for specific model averaging techniques (I think we only have this one so far though).

Reply:
Updated